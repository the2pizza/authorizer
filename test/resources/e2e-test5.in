{"account": {"active-card": true, "available-limit": 100}}
{"transaction": {"merchant": "McDonald's", "amount": 30, "time": "2019-02-13T12:00:00.000Z"}}
{"transaction": {"merchant": "McDonald's", "amount": 30, "time": "2019-02-13T12:01:00.000Z"}}