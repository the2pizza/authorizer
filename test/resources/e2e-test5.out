{"account":{"active-card":true,"available-limit":100},"violations":[]}
{"account":{"active-card":true,"available-limit":70},"violations":[]}
{"account":{"active-card":true,"available-limit":70},"violations":["doubled-transaction"]}